cmd/applyconfig/applyconfig.go

```diff
@@ -588,3 +601,17 @@ func main() {
 
 	logrus.Infof("Success!")
 }
+
+func targetClusterIsDisabled(disabledClusters sets.Set[string]) bool {
```

Comment:
can we just do a `strings.Contains` check here to see if the jobName contains any of the `disabledClusters`? That way, if the job name is changed slightly we won't have to update this code.

Reply:
I did it like it the first time but changed my mind. :) 
Revised.